### DISCUSSION TITLE
Translate one component with multiple users

### DISCUSSION THREAD

**Discussion by infureal** [WeblateOrg/weblate#9164] [Q&A]
Hello, can you tell me, how can we work on one component as a team? 
I mean we using one component for big list of ecommerce product and we need them to be translated. But it hard to do this by one person. But when two or more users click button `[Translate]` they all see the same phrase, but by intuition it need to be different phrases.

**Q. How can two or more users work on one component without correlation?**

**Comment by nijel** [comment]
Currently, the only thing that will happen is that Weblate will refuse to save the string if it has been edited meanwhile.
